products/windows.md

```diff
@@ -72,19 +80,16 @@ releases:
     eol: 2020-01-14
 ---
 
-| Note | Comment                                      |
-| ---- | -------------------------------------------- |
-| (E)  | Enterprise and Education editions            |
-| (W)  | Home, Pro, and Pro for Workstations editions |
-| SAC  | Semi-Annual Channel release                  |
-| LTS  | Long-Term Servicing Channel                  |
+| Note | Comment                                                    |
+| ---- | ---------------------------------------------------------- |
+| (E)  | Enterprise, Education and IoT Enterprise editions          |
+| (W)  | Home, Pro, Pro Education and Pro for Workstations editions |
+| LTS  | Long-Term Servicing Channel                                |
 
+[Windows 11 release information](https://docs.microsoft.com/en-us/windows/release-health/windows11-release-information)  
+[Windows 10 release information](https://docs.microsoft.com/en-us/windows/release-health/release-information)  
 [Microsoft Support Windows Lifecycle Fact Sheet](https://support.microsoft.com/en-in/help/13853/windows-lifecycle-fact-sheet)
 
-Windows 10 Semi-Annual Channel (SAC) is governed by the [Modern Lifecycle Policy][mlp] (as of January 25, 2018).
-
 Feature updates will be released twice a year for Windows 10 via the Semi-Annual Channel (SAC), targeting March and September annually.
```

Comment:
Can we specify this line so people can see the different between  before 21H2  and after21H2.
This one is the last thing before merge so after you change this +1 from me for merge

Reply:
Good point, changed it to the formulation in https://docs.microsoft.com/en-us/windows/release-health/release-information